sdk/core/azure-core-experimental/samples/pyodide_integration/browser.py

```diff
@@ -0,0 +1,105 @@
+from unittest.mock import _patch_dict, patch
+from uuid import uuid4
+
+from azure.ai.textanalytics.aio import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+from azure.core.pipeline.transport import HttpRequest
+from azure.core.experimental.transport import PyodideTransport
+from azure.storage.blob.aio import BlobClient, BlobServiceClient
+
+# pylint: disable=import-error
+from async_testing import AsyncTestSuite
+
+
+class PyodideTransportIntegrationTestSuite(AsyncTestSuite):
```

Comment:
I think we should describe that the sample runs text analytics

Reply:
We have this information in readme.md. Are you suggesting modifying the class name to include it too?